### PR TITLE
Parthabranch

### PR DESCRIPTION
**Pull request by jonnypaul75**
i have changed some files

### CODE REVIEW

File: partha.html [hunk 1]
```diff
@@ -1 +1 @@
-hello world
\ No newline at end of file
+hello world ver 2
```
Comment (ankursoft) [inline partha.html:1]: eta ke ver 20 koro

File: partha.html [hunk 2]
```diff
@@ -0,0 +1 @@
+hello world
```
Comment (ankursoft) [inline partha.html]: this is OK

**Review by ankursoft — CHANGES_REQUESTED** [review]
I have suggested change. Pls do it

File: partha.html (hunk 1 above)
Comment (jonnypaul75) [reply partha.html:1]: i have done

File: partha.html [hunk 3]
```diff
@@ -1 +1 @@
-hello world ver 2
\ No newline at end of file
+hello world ver 20
```
Comment (ankursoft) [inline partha.html:1]: Ok satisfied

**Review by ankursoft — APPROVED** [review]

**Pull request merged**